Create _index.md

indianakretzschmar opened the pull request:
Draft

indianakretzschmar committed:
Create _index.md

cjmyers commented:
Looks like this is not ready for review yet.  I don't see any changes.  Did you remember to checkin your files?

indianakretzschmar committed:
Add author profile for Indiana Kretzschmar

Added detailed author information for Indiana Kretzschmar, including title, role, education, interests, projects, and social links.

indianakretzschmar committed:
Fix email link formatting in author index

indianakretzschmar committed:
Add files via upload

indianakretzschmar committed:
Fix link formatting for LinkedIn profile

Gonza10V reviewed (approved):
look good on deploy
